Regression Discontinuity in Time (RDiT).md

```diff
@@ -0,0 +1,183 @@
+
+{{% codeblock}}
+```R
+#use the with function to select a bandwidth
+#set polynomial order with p=...
+#set number of bins with nbins=...
+#y.lim is used to define the height of the y-axis
+with(subset(fertility, date >= 540 & date <=599), rdplot(y = abortions, p=2, x = month_of_conception, nbins=30, y.lim = c(0, 10000), title="Discontinuity in abortions after the introduction of the benefit", x.label="Month (centered on policy introduction date)", y.label="Abortions"))
+```
+{{% /codeblock}}
+
+We see a clear discontinuity at the treatment date. Now, we can test for this formally by running a standard RDiT regression:
+
+{{% codeblock}}
+```R
+#estimation of treatment effect with a second-order polynomial
+#use natural log of abortions as in the paper
+options(scipen = 999) #remove scientific notation
+fertility$month_qdr <- fertility$month_of_birth^2 #create quadratic of the time variable
+rdit <- lm(ln_abortions ~ month_of_birth + month_qdr + treatment_dummy + month_of_birth*treatment_dummy + month_qdr*treatment_dummy, subset(fertility, date <= 599 & date >= 540)) 
+summary(rdit)
+```
+{{% /codeblock}}
+
+We see a statistically significant negative effect of the treatment on abortions, which is also what Gonzalez (2013) finds in her paper. Now, let's run some of the robustness checks mentioned above. We start by testing for serial correlation in the residuals using the Durbin-Watson test.
+
+{{% codeblock}}
+```R
+#Durbin-Watson test for serial correlation in the residuals
+dwtest(rdit, alternative = "two.sided")
+```
+{{% /codeblock}}
+
+We see that the p-value is well below 0.05, meaning we can reject the null hypothesis of no serial correlation. Given that residuals exhibit this serial correlation, we compute HAC standard errors:
+
+{{% codeblock}}
+```R
+#use vcovHAC in coeftest to re-estimate the model with HAC standard errors
+coeftest(rdit, vcovHAC(rdit))
+```
+{{% /codeblock}}
+
+The HAC standard errors do differ from the conventional ones, but do not change the inference we made from the original estimates. The treatment coefficient remains strongly statistically significant. 
+
+Now, we estimate a donut RDD, allowing for the possibility that individuals anticipate the policy and change their behaviour accordingly. We assume that they can do so within 1 month of the treatment date (before and after).

```

Comment:
It may be nice to talk a bit more extensively about the donut RDD and refer to [Barreca et al., 2011](https://www.sciencedirect.com/science/article/pii/S0047272722000627#b0015) . [Gerfin et al. (2015)](https://www.sciencedirect.com/science/article/pii/S0047272722000627#b0080) is an example where this has been used.